client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -1844,8 +1848,8 @@ public void testIndexPutSettings() throws Exception {
         {
             // tag::put-settings-settings-source
             request.settings(
-                    "{\"index.number_of_replicas\": \"2\"}"
-                    , XContentType.JSON); // <1>
+                "{\"index.number_of_replicas\": \"2\"}"
```

Comment:
Can you revert all of these formatting changes above?

Reply:
I think I messed up something during the merge from master and my editor did all these formats. I will revert everything back.